schema.sql

```diff
@@ -1,5 +1,4 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(id INT GENERATED ALWAYS AS IDENTITY,name VARCHAR NOT NULL,date_of_birth DATE NOT NULL,escape_attempts INT NOT NULL,neutered 
+BOOL NOT NULL,weight_kg DECIMAL(50,2) NOT NULL);
```

Comment:
- [ ] **[OPTIONAL]** I suggest you mark the `id` column as the primary key. A primary key constraint indicates that a column, or group of columns, can be used as a unique identifier for rows in the table. This requires that the values be both unique and not null.

Reply:
Thank you for the feedback @julie-ify  I will work on that